content/Recordings.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Recordings
+---
```

Comment:
Follow the default [template](https://github.com/p-society/psoc-website-new/blob/master/archetypes/default.md) (it may seem irrelevant now, but is [useful](https://gohugo.io/content-management/archetypes/) when migrating to a new theme or performing a hugo specific command).

> A good way to ensure you're following the archetype is to use `hugo new <path-to-the-file.md>`